image/patches/anonymize_user.patch

```diff
@@ -0,0 +1,16 @@
+--- a/lib/tasks/users.rake
++++ b/lib/tasks/users.rake
```

Comment:
Is there any other way to do this? We had something similar for Indico for creating an admin user. Is there a spec for this change?

Reply:
It's possible to do it making a couple of API calls, one to retrieve the user id from the username and another one to anonymize, however, there are two reasons to prefer this approach:
* It is way simpler to leverage the rake tasks
* Although patching the code is far from ideal, this approach is already in place for fix https://github.com/discourse/discourse/pull/20523
* The patch is temporary, it will be removed when the https://github.com/discourse/discourse/pull/20522 is merged and Discourse, upgraded